api/db_connection.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+async function connect() {
+  try {
+    const connection = await mongoose.connect(
+      "mongodb+srv://admin:admin@bci-mongodb.ru25h.mongodb.net/BCI-MongoDB?retryWrites=true&w=majority"
```

Comment:
Keep it secret in env and remove from SCM asap

Reply:
Changed